dialects/tests/test1.yaml

```
@@ -0,0 +1,35 @@
+name: testdb
+dependencies:
+  arithmetic: 
+    https://github.com/substrait-io/substrait/blob/main/extensions/functions_arithmetic.yaml
+  comparison: 
+    https://github.com/substrait-io/substrait/blob/main/extensions/functions_comparison.yaml
+supported_types:
+  i32:
+    sql_type_name: TINYINT
+    supported_as_column: true
+  i64:
+    sql_type_name: TINYINT
+    supported_as_column: true
+scalar_functions:
+- name: comparison.is_null
+  local_name: IS NULL
+  notation: POSTFIX
+  supported_kernels:
```

the extensions files call these `impls`, if I understood right what this means? I don't think there is anything in Substrait called "kernels"

```suggestion
  supported_impls:
```

---

Done.